### PR TITLE
#173 Add auto complete to telephone and email entries

### PR DESCRIPTION
**Pull request by agnieszkazbikowska-madetech**

### CODE REVIEW

**Comment by agnieszkazbikowska-madetech** [comment]
[Add auto complete to telephone and email entries](https://trello.com/c/9ER8rlw5/173-add-auto-complete-to-telephone-and-email-entries)

File: tests/unit/components/textInput.test.js [hunk 1]
```diff
@@ -107,8 +97,22 @@ describe('textInput', () => {
     expect(container.querySelector(`[data-testid="${textInputName}"]`).value).toBe(value);
   })
 
+  test('Displays autoComplete', () => {
```
Comment (Bhavesh-Patel) [inline tests/unit/components/textInput.test.js:100]: This is testing an attribute, not that something is display - please update the test title
```suggestion
  test('Text input has correct autoComplete attribute', () => {
```

File: compoments/radioFieldSet.js [hunk 2]
```diff
@@ -177,6 +191,4 @@ RadioFieldSet.propTypes = {
   errorText: PropTypes.string,
   conditionalValue: PropTypes.object
 };
-export default RadioFieldSet;
-
-
+export default RadioFieldSet;
```
Comment (Bhavesh-Patel) [inline compoments/radioFieldSet.js:194]: Please restore the new line at the end of the file

**Review by Bhavesh-Patel — CHANGES_REQUESTED** [review]
A couple of really small changes

File: compoments/radioFieldSet.js (hunk 2 above)
Comment (agnieszkazbikowska-madetech) [reply compoments/radioFieldSet.js]: All done :-) .

File: compoments/report-repair/contact-details.js [hunk 3]
```diff
@@ -20,18 +20,31 @@ const ContactDetails = ({handleChange, values}) => {
...
-      label: 'Please enter your email address',
-      type: 'email', validator: emailValidator,
-      emptyInputErrorMessage: 'Enter an email address',
-      invalidInputErrorMessage: 'Enter a valid email address',
-    }}
+    {
+      value: 'text',
+      title: 'Text message (recommended)',
+      conditional: {
+        autoComplete: 'tel-national',
```
Comment (agnieszkazbikowska-madetech) [inline compoments/report-repair/contact-details.js:27]: @Bhavesh-Patel I've found this example https://design-system.service.gov.uk/patterns/telephone-numbers/ just now and it's using `autocomplete="tel"`. Do you think I should replace to match the design systems?

Comment (agnieszkazbikowska-madetech) [reply compoments/report-repair/contact-details.js]: All done!

**Review by Bhavesh-Patel — APPROVED** [review]

**Pull request merged**